airflow/configuration.py

```diff
@@ -1548,9 +1548,8 @@ def initialize_config() -> AirflowConfigParser:
         if local_conf.getboolean("core", "unit_test_mode"):
             local_conf.load_test_config()
 
-    # Make it no longer a proxy variable, just set it to an actual string
     global WEBSERVER_CONFIG
```

Comment:
Let's move this up to the list of globals at the top of the function.

Reply:
Made the changes